best-practices.md

```diff
@@ -297,6 +298,51 @@ is strongly recommended that Catalogs don't contain differently versioned Items
 consistent (Sub-)Catalogs containing either all or no data. Collections that are referenced from Items should always use the same
 STAC version. Otherwise some behaviour of functionality may be unpredictable (e.g. merging common fields into Items or reading summaries).
 
+## Using Summaries in Collections
+
+One of the strongest recommendations for STAC is to always provide [summaries](collection-spec/collection-spec.md#summaries) in
+your collections. The core team decided to not require them, in case there are future situations where providing a summary
+is too difficult. The idea behind them is not to exhaustively summarize every single field in the collection, but to provide
+a bit of a 'curated' view. 
+
+Some general thinking on what to summarize is as follows:
+
+* Any field that is a range of data (like numbers or dates) is a great candidate to summarize, to give people a sense what values
+the data might be. For example in overhead imagery, a 
+[`view:off_nadir`](extensions/view/README.md#item-properties-and-item-asset-fields) with a range of 0 to 3 would tell people this 
+imagery is all pretty much straight down, while a value of 15 to 40 would tell them that it's oblique imagery, or 0 to 60 that it's 
+a collection with lots of different look angles. 
+
+* Fields that have only one or a handful of values are also great to summarize. Collections with a single satellite may
+use a single [`gsd`](item-spec/common-metadata.md#instrument) field in the summary, and it's quite useful for users to know
+that all data is going to be the same resolution. Similarly it's useful to know the names of all the 
+[`platform` values](item-spec/common-metadata.md#instrument) that are used in the collection. 
+
+* It is less useful to summarize fields that have numerous different discrete values that can't easily be represented
+in a range. These will mostly be string values, when there aren't just a handful of options. For example if you had a 
+'location' field that gave 3 levels of administrative region (like 'San Francisco, California, United States') to help people
+understand more intuitively where a shot was taken. If your collection has millions of items, or even hundreds, you don't want
+to include all those values as a summary. 
+
+* Fields that consist of arrays are more of a judgement call. For example [`instruments`](item-spec/common-metadata.md#instrument)
```

Comment:
Maybe add eo:bands, too? That's the second most prominent example and probably one that people would think about.

Reply:
I had thought that having the following paragraph talk explicitly about eo:bands was sufficient, as I think people will read the whole section if they are actually thinking about it. But can add here too if you think it's important.

Reply:
I just realized we have a "false" statement here: The collection spec says:

> If the field summarizes an array (e.g. instruments), the field's array elements of each Item must be merged to a single array with unique elements.

So it's not "a judgement call" at the moment. The current spec basically don't work for data types that use arrays with some hidden sematics, like "proj:shape", where it feels like the structure should be object as it's not really a list.

Reply:
I've looked into the arrays we define in extensions as new field and except proj, this requirement works for all arrays, I think. I'm not 100% sure about label extension, but it seems okay. So it's just proj, but for those fields I'd say (1) they should not have been defined as arrays - they are a better fit for an object - and (2) they shouldn't get summarized anyway.

Reply:
I think I understand, and agree with the logic but what's the next step? What changes do I need to make on this PR?

Reply:
I've read it again and indeed there's nothing to do here, I had understood the "judgement call" differently before.